sphinx_docs/conf.py

```diff
@@ -31,9 +31,9 @@
 ######################################################
 
 # General information about your project.
-project = "Tableread - turn a rST table into a Python object!"
-copyright = "2018, Brad Brown"  # noqa
-author = "Brad Brown"
+project = "layered-config - library for layering config files"
+copyright = "2018, Doug Philips and Ryan Casperson"  # noqa
+author = "Doug Philips"
```

Comment:
Go ahead and update the `author` field in `pyproject.toml` to match this related change. Same thing, no need to keep me in that authors list. I'll take the honorary, undocumented title of Editor in Chief 😄.

Reply:
Ok.